common.js

```diff
@@ -1708,6 +1708,63 @@ const allSoloCars = {
       'all': ['ssp'],
     },
   },
+  'Oldsmobile': {
+    'Calais W41': {
+      'all': ['hs'],
+    },
+    'Starfire (V6)': {
+      'all': ['esp'],
+    },
+    'Sunbird (V6)': {
+      'all': ['esp'],
+    },
+    'Omega': {
+      'all': ['f'],
```

Comment:
This is not a valid class. Did you mean fsp?

Reply:
yes

fixed it